roles/passenger/README.md

```diff
@@ -0,0 +1,57 @@
+# Passenger
+
+Borrowed from [pulibrary / princeton_ansible](https://github.com/pulibrary/princeton_ansible/tree/main/roles/passenger)
+
+Originally borrowed from Geerling Guy
+
+Installs Passenger (with Nginx) Ubuntu linux servers.
+
+## Requirements
+
+None.
+
+## Role Variables
+
+Available variables are listed below, along with default values (see `defaults/main.yml`):
+
+    passenger_server_name: www.example.com
+
+The server name (used in the Nginx virtual host configuration).
+
+    passenger_app_root: /opt/example/public
+
+The `passenger_root` for your application (e.g. the `public` folder in a rails app).
+
+    passenger_app_env: production
+
+The app environment passenger will load.
+
+    passenger_root: /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini
+    passenger_ruby: /usr/local/bin/ruby
+
+Values for passenger configuration directives inside `nginx.conf`. These defaults should generally work correctly, but if you build `ruby` on its own (as an example), the path to ruby may be different.
+
+    nginx_worker_processes: "{{ ansible_processor_vcpus | default(ansible_processor_count) }}"
+    nginx_worker_connections: "768"
+    nginx_keepalive_timeout: "65"
+    nginx_remove_default_vhost: true
+
+Nginx directives.
+
+## Dependencies
+
+None.
+
+## Example Playbook
+
+    - hosts: server
+      roles:
+        - { role: passenger }
+
+## License
+
+MIT / BSD
+
```

Comment:
I took out `LICENSE` here entirely since it's also in the meta `main.yml`. Idk why the role template has so much duplication tbh